scripts/build.ps1

```diff
@@ -42,12 +44,34 @@ if (!(Test-Path "dotnetcli\dotnet.exe")) {
         Write-Error "Failed to install framework version 2.0.0, exit code [$lastexitcode], aborting build."
         exit -1
     }
+} else {
+    Write-Host "dotnet.exe is installed, checking for latest."
+
+    $cliVersion = Invoke-Expression "$dotnetExePath --version"
+    if ($cliVersion -ne $Version) {
+        Write-Host "Newest version of dotnet cli not installed, downloading and installing."
+        Invoke-Expression -Command "$PSScriptRoot\install-dotnet.ps1 -Channel master -Version $cliVersion -InstallDir $PSScriptRoot\..\dotnetcli"
```

Comment:
`-Channel master -Version $cliVersion`

This needs to be `-Channel master -Version $Version`

Reply:
Doh! Thanks.